elastic/kibana
# [Infrastructure UI] Host View: Store pagination and sorting info in the URL state

Closes #146583

## Summary

This PR adds table sorting and pagination to the url state. It uses a separate reducer for that because that way we can avoid expanding the other state and will avoid rerendering on change.

## Testing
1. Open the host view
2. Select different rows per page and different sorting column
3. Copy the URL and paste it into a different browser tab - the table sorting, rows per page, and current page should be persisted
<img width="1726" alt="image" src="https://user-images.githubusercontent.com/14139027/205620477-3042a2ea-48b4-4cd8-8f18-b62cb6133438.png">
